x-pack/plugins/ml/public/application/data_frame_analytics/pages/job_map/page.tsx

```diff
@@ -120,7 +120,7 @@ export const Page: FC = () => {
         </MlPageHeader>
       ) : null}
       {jobId !== undefined ? (
```

Comment:
I think this  check on 122 needs to be `jobId !== undefined && mapModelId === undefined` since when we set `jobId` and `modelId` we give it a default value. That's why we're seeing double headings. 

The one on 133 should be `modelId !== undefined && mapJobId === undefined ` for the same reason.